docs/index.md

```diff
@@ -32,12 +32,13 @@ For integration with specific providers like OpenAI, additional dependencies can
 pip install ecologits[openai]
 ```
 
-EcoLogits currently supports the following providers:
+EcoLogits currently supports the following providers/clients:
```

Comment:
@samuelrince As we now support clients such as HF or LiteLLM above providers, we should maybe be clearer in the doc

Reply:
Great idea 👌